db/app/app-tests/app.test.js

```diff
@@ -50,3 +50,40 @@ describe("GET /api", () => {
       });
   });
 });
+
+describe('/api/articles/:article_id', () => {
+  test('200: should return article data for specific article_id provided', () => {
+    return request(app)
+    .get("/api/articles/1")
+    .expect(200)
+    .then(({body}) => {
+      expect(body).toEqual([{
+        article_id: 1,
+        title: "Living in the shadow of a great man",
+        topic: "mitch",
+        author: "butter_bridge",
+        body: "I find this existence challenging",
+        created_at: expect.any(String), // the Number from articles.js keeps converting to an actual string data - hopefully this is correct
+        votes: 100,
+        article_img_url:
+          "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+      }])
+    })
+  });
+  test('404: returns error for invalid article id NUMBER', () => {
+    return request(app)
+    .get('/api/articles/124124')
+    .expect(404)
+    .then(({body}) => {
+      expect(body.msg).toEqual("invalid article id")
+    })
+  })
```

Comment:
Excellent test but the test description is not correct here, this is a valid article id, but it simply does not exist, which is why we are expecting a 404 here.